### ISSUE TITLE
Refresh: audit new components with large font settings

### ISSUE THREAD

**Issue by maureenlholland** [mozilla/bedrock#15635]
## Description

Browse the new design with large font setting and note anything that looks off

Tested cross-browser with the following components:
- banner
- top nav
- sub nav
- mobile nav
- launchpad
- gallery
- springboard
- newsletter container
- footer primary
- footer secondary
- lang switcher
- feature
- 4 column layout theme
- showcase

## Must fix
- Springboard overflow on Firefox around 1024px
<img width="1042" alt="firefox-1024px-overflow" src="https://github.com/user-attachments/assets/98524c1b-c3f6-406c-89b9-7313135bbaaf">
- Lack of footer gap resolved in https://github.com/mozilla/bedrock/pull/15698

## Nice to have 
- give more space to the springboard preview text (around 770px chrome)
<img width="494" alt="769px-springboard" src="https://github.com/user-attachments/assets/d3a2ca01-7fd6-4367-9cd9-158f6def2760">

- improve spacing/content width on top nav (around 770px safari)
<img width="771" alt="769px-links" src="https://github.com/user-attachments/assets/e58f3363-bd67-4c70-9e8c-037026936ef0">

- improve mobile menu vertical alignment
<img width="395" alt="vertical-alignment" src="https://github.com/user-attachments/assets/c7908673-5bc6-4e8d-92e5-912acf097bb6">

---

#### Success Criteria
- [x] All new components have been audited
- [ ] Springboard crunching / overflow is fixed
- [ ] No underline overlap on nav menu button
- [ ] Better spacing and alignment on top nav

**Comment by maureenlholland** [comment]
Footer could use gap
<img width="482" alt="Screenshot 2024-12-03 at 12 26 51 PM" src="https://github.com/user-attachments/assets/223813df-c501-4ea6-9b6d-305bdb4c0859">

Menu button after element underline needs adjustment
<img width="326" alt="Screenshot 2024-12-03 at 12 25 21 PM" src="https://github.com/user-attachments/assets/4a07904d-9fbd-4802-b753-20a258605035">

**Comment by maureenlholland** [comment]
@wen-2018 I updated the comment with what I saw after more thorough testing. The springboard overflow is still top issue for me.

**Comment by wen-2018** [comment]
@maureenlholland Okay, I will separate PRs for each component fix
